src/rdo.rs

```diff
@@ -813,6 +837,12 @@ pub fn rdo_tx_type_decision(
 
   let cw_checkpoint = cw.checkpoint();
 
+  let rdo_type = if fi.use_tx_domain_distortion {
+    RDOType::TxDist
+  } else {
+    RDOType::Full
```

Comment:
So, 'Full' means pixel domain not both tx and pixel, correct?

Reply:
It means Pixel Domain + Full Rate. I can come up with different names or comments. Two other types I want to add include:

TxDistRate = tx_dist plus tx_dist_rate
MetricEst = SATD based rate and distortion estimate

I also need a training mode. I could put that as a RDOType or just have a separate flag for it.